### PR TITLE
Handle `create` and `drop` commands in Hana handler

### PR DESCRIPTION
**Pull request by polyal**
# Description

Closes #{issue_number}.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [ ] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Comment by AlekSi** [comment]
@polyal Please sign our CLA by clicking a button in that comment: https://github.com/FerretDB/FerretDB/pull/2458#issuecomment-1511950521

File: internal/handlers/hana/hanadb/collections.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hanadb
+
+import (
+	"context"
+	"fmt"
+
+	_ "github.com/SAP/go-hdb/driver"
```
Comment (AlekSi) [inline internal/handlers/hana/hanadb/collections.go:21]: It is not needed in that file

File: internal/handlers/hana/hanadb/collections.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+// CreateCollection creates a new SAP HANA JSON Document Store collection.
+//
+// It returns ErrAlreadyExist if collection already exist.
+func (hanaPool *Pool) CreateCollection(ctx context.Context, db, collection string) error {
+	sql := fmt.Sprintf("CREATE COLLECTION \"%s\".\"%s\"", db, collection)
```
Comment (AlekSi) [inline internal/handlers/hana/hanadb/collections.go:28]:
```suggestion
	sql := fmt.Sprintf("CREATE COLLECTION %q.%q", db, collection)
```

And let's do the same in other places

File: internal/handlers/hana/hanadb/databases.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hanadb
+
+import (
+	"context"
+	"fmt"
+
+	_ "github.com/SAP/go-hdb/driver"
```
Comment (AlekSi) [inline internal/handlers/hana/hanadb/databases.go:21]: Also not needed

File: internal/handlers/hana/hanadb/hanadb.go [hunk 4]
```diff
@@ -27,12 +32,34 @@ var (
...
+
+// Errors from HanaDB.
+var Errors = map[int]error{
+	259: ErrTableNotExist,
+	288: ErrCollectionAlreadyExist,
+	362: ErrSchemaNotExist,
+	386: ErrSchemaAlreadyExist,
+}
+
+func getHanaErrorIfExists(err error) error {
```
Comment (AlekSi) [inline internal/handlers/hana/hanadb/hanadb.go:56]: We typically add comments to all top-level declarations, not only to exported

**Review by AlekSi — COMMENTED** [review]
LGTM overall, just a few comments

**Review by AlekSi — APPROVED** [review]

**Review by chilagrow — COMMENTED** [review]
Great work. 

I'm seeing linter warning, could you fix that please :hugs: 

```
bin/task lint
task: [lint] bin/golangci-lint run --config=.golangci.yml
Error: internal/handlers/hana/hanadb/hanadb.go:58:96: Comment should end in a period (godot)
task: Failed to run task "lint": exit status 1
task: Failed to run task "lint": exit status 1
// Returns one of the errors above or the original error if the formatted version doesn't exist
                                                                                               ^
Error: Process completed with exit code 201.
```

**Review by chilagrow — APPROVED** [review]

**Review by noisersup — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]
LGTM 👍

**Pull request merged**